content/02_general-rules/04_asset-modifications.md

```diff
@@ -1,10 +1,63 @@
 ### Asset Modifications
 
-Texture modifications are allowed, with some caveats. The textures of portal surfaces
-must not be modified in any way. No textures can be modified to provide any visual
-assistance, such as help with a lineup. No texture may be modified to be more
-translucent than it was, with the exception of the player's viewmodel.
-
-Modifying models is not allowed, with the exception of the portal gun (including both
-the viewmodel and the in-world model). Modifying audio files, captions or any other
-asset is not allowed.
+The asset modification rules will be split into their respective categories that are 
+seen when looking at the main pak01_dir.vpk in the game’s files. There will also be 
+a “General Rules” category that applies to any kind of asset modification.
+
+#### General Rules
+
+Any asset modifications are subject to being grounds for the rejection or removal of 
+any run on speedrun.com or the cm boards.
+
+When assets are required to be submitted alongside a run, these assets must adhere to 
+speedrun.com TOS and any other applicable TOS for wherever your proof of the run is 
+being stored. Including video submissions.
+
+Editing .dll files that are present in the files to modify an asset is not allowed.
+
+If you have any questions about whether something is allowed or not, please ask.
+
+#### Textures
+
+Portalable surfaces cannot be modified for any reason.
+
+No textures can be modified to provide any visual assistance, such as help with a lineup.
+
+Textures can not be more transparent than their original texture, this tends to be seen 
+most in light bridge modifications. The only exception to this rule is the player’s 
+viewmodel for both the portal gun and PotatOS.
+
+#### Models
+
+Modifying models of assets is not allowed, except for both the world model and the 
+viewmodel of the portal gun. The world model of the portal gun cannot be used to gain an 
+advantage for any reason.
+
+#### Particles
+
+Particles can be changed only if it doesn’t require editing any of the game’s .dll files.
+
+Particle changes cannot be used for an advantage, nor can they be entirely removed.
+
+#### Resource
+
+Resource files can be changed, but not to gain an advantage or to outright remove them.
+
+Any resource file that changes how the menu is structured, or how they function is not 
+allowed to be active during singleplayer SLA runs.
```

Comment:
Category-specific rules should go in the relevant category. Remove this section, and in the any% extra rules section write something like "The structure and functionality of game menus may not be modified in any capacity, by modifying resource files or otherwise."